packages/ipfs-multipart/src/parser.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const Content = require('@hapi/content')
-const multipart = require('it-multipart')
+const multipart = require('./it-multipart')
```

Comment:
Context on this change https://github.com/ipfs/js-ipfs/pull/2874/files#r389970339